util/log/clog.go

```diff
@@ -37,6 +37,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/util"
```

Comment:
`log` already depends on `util`; i found out the hard way when I attempted to to have the functionality in `log`